### ISSUE TITLE
Error building any example after building a different one

### ISSUE THREAD

**Issue by dbandstra** [dbandstra/zang#39]
At first you can build any example, but if you try to build any other example after that, it will fail with this message:

```
error: FileNotFound
arpeggiator...The following command exited with error code 1:
/Users/dbandstra/zig-macos-x86_64-0.7.0/zig build-exe /Users/dbandstra/code/zang/examples/example.zig --library SDL2 --library c --pkg-begin build_options /Users/dbandstra/code/zang/zig-cache/arpeggiator_build_options.zig --pkg-end --cache-dir /Users/dbandstra/code/zang/zig-cache --name arpeggiator --pkg-begin wav /Users/dbandstra/code/zang/examples/zig-wav/wav.zig --pkg-end --pkg-begin zang /Users/dbandstra/code/zang/src/zang.zig --pkg-end --pkg-begin zang-12tet /Users/dbandstra/code/zang/src/zang-12tet.zig --pkg-end --pkg-begin zangscript /Users/dbandstra/code/zang/src/zangscript.zig --pkg-end -I /usr/local/include/SDL2 -L /usr/local/lib -D _THREAD_SAFE --enable-cache
error: the following build command failed with exit code 1:
/Users/dbandstra/code/zang/zig-cache/o/9ef725a5da104ba121029526507e6d3a/build /Users/dbandstra/zig-macos-x86_64-0.7.0/zig /Users/dbandstra/code/zang /Users/dbandstra/code/zang/zig-cache arpeggiator
```

If you delete `zig-cache` you can then build a different example.

This seems to be a regression in Zig 0.7.0.

**Comment by dbandstra** [comment]
Filed https://github.com/ziglang/zig/issues/7228

**Comment by dbandstra** [comment]
Fixed in Zig master, will be in the 0.7.1 release
